CONTRIBUTING.md

```diff
@@ -23,3 +23,21 @@ changes, feel free to ping @gwyneplaine
 * Please [follow our established coding conventions](https://github.com/keystonejs/keystone/wiki/Coding-Standards)
 (with regards to formatting, etc)
 * All new features and changes need documentation.
+
+## Running the docs website
+
+You can run the website in a local environment using the following steps:
+
+```sh
+git clone https://github.com/JedWatson/react-select
+cd react-select
+yarn
+yarn build
+cd docs
```

Comment:
This feels wrong to me, since the script itself is described as
```"start": "cd docs && yarn start",```
It does the `cd` part

Reply:
That bit actually feels a bit unnatural to me. As someone who visited the repository for the first time, I saw a `package.json` inside the docs folder and assumed the website's an isolated folder, which it is too. The "start" command at the root level's just an abstraction. 

I didn't even know going to the root folder's required for running docs until the discussion in #4232

But anything's fine here I guess. Should I change the commands to the below code?

```sh
cd react-select
yarn
yarn build
yarn start
```

Reply:
This looks good @cseas, could you please change the commands and we'll look at getting this merged in.

Reply:
I don't think you even need to run `yarn build` since `preconstruct dev` is run as a `postinstall` script. I believe it can just be:

```
cd react-select
yarn install
yarn start
```